app/assets/javascripts/govuk_publishing_components/components/feedback.js

```diff
@@ -49,6 +50,22 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
         revealInitialPrompt()
       })
 
+      this.$pageIsNotUsefulButton.on('click', function (e) {
+        var cookie = document.cookie.split("; ") ;
+        var i ;
+        var clientIDFound = false;
+        for (i = 0; i < cookie.length; i++) {
+          if (cookie[i].startsWith("_ga")) {
```

Comment:
`.startsWith` is an [ES6 feature](http://kangax.github.io/compat-table/es6/#test-String.prototype.startsWith), which unfortunately is not supported in the [browsers we need to support](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices).

You'll need to use `indexOf`, which is widely supported.

e.g. `if (cookie[i].indexOf('_ga') !== -1)`

Reply:
I've addressed a lot of these comments by using `window.GOVUK.cookie('_ga')` rather than this funky iteration so will resolve them.